Userland/Libraries/LibAudio/Buffer.h

```diff
@@ -115,6 +113,14 @@ class Buffer : public RefCounted<Buffer> {
     {
     }
 
+    // Empty Buffer representation, to avoid tiny anonymous buffers in EOF states
+    Buffer()
+        : m_buffer()
+        , m_id(-1)
+        , m_sample_count(0)
```

Comment:
This seems like the plain types should be initialized at the declaration:
```c++
    const i32 m_id { -1 };
    const int m_sample_count { 0 };
```

Then this ctor can just be:
```c++
Buffer() = default;
```

Reply:
oh, yep, that's a cleaner solution. Using = default now.